ui/src/Styles/TagRowsContainer.scss

```diff
@@ -15,8 +15,8 @@
  * limitations under the License.
  */
 
-@import '../Styles/Main.scss';
-@import '../Styles/Breakpoints.scss';
+@import 'Main';
+@import 'Breakpoints';
```

Comment:
Interesting! I assume compiler looks for the file name in entire project to find the correct one to import?